english/dream/pancakes/make-a-pancake.md

```diff
@@ -0,0 +1,2 @@
+If you want you can make a pancake with your special marshmallow cat. You can make nutella crepes, or big fluffy pancakes with maple syrup.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.